operator.mdx

```diff
@@ -1,207 +1,53 @@
-* Support for multi-region data distrubtion (geo-replication)
-* Support for multi-region writes (multi-master)
-* Option to pick Serverless throughput or Provisioned throughput
-  * Serverless throughput is consumption-based and charges for resources used by database operations, no minimum cost
-    * Best for worklods with low traffic, occasional spikes/bursts, and moderate performance requirements
-  * Provisioned throughput offers guaranteed speed and availability, but requires planning and managing of throughput capacity
-    * Best for large, high-throughput workloads with high performance requirements
-    * Customer can also override provisioned throughput with autoscale throughput via the collection/container setup
- * Option to enable automatic backups to protect account
-
-#### Security 
-
-* By default Cosmos DB applies encryption at rest automatically
-  * A second layer of encryption with CMK is available, but would need additional terraform configuration to support
-  * Sometimes regenerating new keys may be necessary, is there a way we can support that? If not, we can just set up a guide for how to do that within the Azure portal
-
-#### Monitoring
-
-* Potential metrics to monitor:
-  * Mongo requests
-    * Number of Mongo requests made
-  * Mongo requests units charged
-    * Mongo RUs consumed
-  * Server side latency
-  * Normalized RU consumption
-    * Max RU consumption percentage per minute
-  * Region onlined
-    * If one of the regions used by the account becomes available
-  * Region offlined
-    * If one of the regions used by the account goes down
-  * Data usage 
-    * Total data usage
-
-### Trade-offs
-
-* Regions not supported in Cosmos DB
-  * East US 3
-* Regions that don't support geo location zone redundancy
-  * North Central US
-  * West Central US
-  * West US
-* Regions that don't support zone redundant Backups
-  * West US
-* Regions in high demand that require a support ticket request for access
-  * Central US
-  * East US
-  * East US 2
-  * West Central US
-  * West US 2
-* Mongo server version 3.2 is not available as a downgrade option, so it's been removed as a param.
-
-### Permissions
-
-Contributor permissions should be all that is necessary to provision the Resource Group and Cosmos DB account.
+# azure-cosmosdb-mongo
+Azure Cosmos DB is a fully managed NoSQL and relational database for modern app development. Azure Cosmos DB offers single-digit millisecond response times, automatic and instant scalability, along with guarantee speed at any scale. Business continuity is assured with SLA-backed availability and enterprise-grade security.
+
+## Use Cases 
+As a fully managed service, Azure Cosmos DB takes database administration off your hands with automatic management, updates and patching. It also handles capacity management with cost-effective serverless and automatic scaling options that respond to application needs to match capacity with demand.
+### Cloud applications
```

Comment:
I'd add a mongo-specific use case. I know with cosmodb you choose SQL or Mongo so these guides will be very similar. A mongo use-case could help someone pick between the two.

Reply:
Good suggestion